dev-util/lcov/lcov-1.14_p2.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LCOV_V=${PV/_p*/}
+DB_V=${PV/*_p/}
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/linux-test-project/lcov.git"
+	inherit git-r3
+else
+	SRC_URI="
+		mirror://sourceforge/ltp/${PN}-${LCOV_V}.tar.gz
```

Comment:
Oh right, I dont remember whether this was killed or not. 
@mgorny any help here?